stubs/migration.stub

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up(): void
+    {
+        Schema::create('{{ tableName }}', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down(): void
+    {
+        Schema::dropIfExists('{{ tableName }}');
+    }
+};
```

Comment:
can you introduce this stub with named argument?

Reply:
Like `create(table: '{{ tableName }}')` and `dropIfExists(table: '{{ tableName }}')` ?

Reply:
yep, good catch mate

Reply:
It's done!